src/proto/tagpost.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package tagpost;
+
+option java_multiple_files = true;
+option java_package = "com.google.tagpost";
+option java_outer_classname = "TagPostProto";
+
+service TagpostService {
+  rpc SimpleReply(SimpleReplyRequest) returns (SimpleReplyResponse) {}
```

Comment:
This name is not descriptive. A name should describe what this RPC is doing, not the complexity of the call.

Reply:
I found it difficult to come up with the right naming here. Any suggestions on the naming of this RPC?